.github/workflows/ci_test-conda.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       fail-fast: false
       matrix:
         # os: [ubuntu-20.04]
-        python-version: [3.7]
+        python-version: [3.6]
```

Comment:
Why revert to 3.6 ?

Reply:
bs it was failing on 3.7 for some strange reason on the docker image

Reply:
I ll check it later... but it is fine for this PR